### ISSUE TITLE
Error processing empty nodes with ID 0.1

### ISSUE THREAD

**Issue by jmnybl** [udapi/udapi-python#53]
Udapi seems to silently delete a sentence-initial empty node (ID 0.1).

Input data (a sentence from UD_Finnish-TDT):
```
# text = Sekaan 1 purkillinen kermaviiliä ja ja sitten jaoin perustahnan 4 lautaselle.
# sent_id =  b712.11
0.1	Laitoin	laittaa	VERB	_	Mood=Ind|Number=Sing|Person=1|Tense=Past|VerbForm=Fin|Voice=Act	_	_	_	_
1	Sekaan	sekaan	ADV	Adv	_	3	orphan	0.1:advmod	_
2	1	1	NUM	Num	NumType=Card	3	nummod	_	_
3	purkillinen	purkillinen	NOUN	N	Case=Nom|Number=Sing	0	root	0.1:obj	_
4	kermaviiliä	kerma#viili	NOUN	N	Case=Par|Number=Sing	3	nmod	_	_
5	ja	ja	CCONJ	C	_	8	cc	_	_
6	ja	ja	CCONJ	C	_	8	cc	_	_
7	sitten	sitten	ADV	Adv	_	8	advmod	_	_
8	jaoin	jakaa	VERB	V	Mood=Ind|Number=Sing|Person=1|Tense=Past|VerbForm=Fin|Voice=Act	3	conj	0.1:conj	_
9	perustahnan	perus#tahna	NOUN	N	Case=Gen|Number=Sing	8	obj	_	_
10	4	4	NUM	Num	NumType=Card	11	nummod	_	_
11	lautaselle	lautanen	NOUN	N	Case=All|Number=Sing	8	obl	_	SpaceAfter=No
12	.	.	PUNCT	Punct	_	3	punct	0.1:punct	_

``` 

Output of a command `cat sentence.conllu | udapy write.Conllu`:
```
# sent_id = b712.11
# text = Sekaan 1 purkillinen kermaviiliä ja ja sitten jaoin perustahnan 4 lautaselle.
1	Sekaan	sekaan	ADV	Adv	_	3	orphan	0.1:advmod	_
2	1	1	NUM	Num	NumType=Card	3	nummod	_	_
3	purkillinen	purkillinen	NOUN	N	Case=Nom|Number=Sing	0	root	0.1:obj	_
4	kermaviiliä	kerma#viili	NOUN	N	Case=Par|Number=Sing	3	nmod	_	_
5	ja	ja	CCONJ	C	_	8	cc	_	_
6	ja	ja	CCONJ	C	_	8	cc	_	_
7	sitten	sitten	ADV	Adv	_	8	advmod	_	_
8	jaoin	jakaa	VERB	V	Mood=Ind|Number=Sing|Person=1|Tense=Past|VerbForm=Fin|Voice=Act	3	conj	0.1:conj	_
9	perustahnan	perus#tahna	NOUN	N	Case=Gen|Number=Sing	8	obj	_	_
10	4	4	NUM	Num	NumType=Card	11	nummod	_	_
11	lautaselle	lautanen	NOUN	N	Case=All|Number=Sing	8	obl	_	SpaceAfter=No
12	.	.	PUNCT	Punct	_	3	punct	0.1:punct	_

```

Udapi is installed with a command: `pip3 install udapi`

**Comment by martinpopel** [comment]
Thanks for reporting this bug.  It should be fixed in 7298c29. I'll deploy a new version to PyPI once I fixed other bugs.
The support for enhanced dependencies in Udapi is far from ideal - I just did not need it now and don't have time for working on something I don't need.:-)

BTW: According to the UDv2 guidelines, if you decide to use enhanced dependencies, you should copy all the basic dependencies to the DEPS columns except for the edges which you want to delete in the enhanced deps graph. It seems that you don't follow this rule and just add the edges which are enhanced-only. (Personally, I like you approach better than the current guidelines, but that is another story.)

**Comment by jmnybl** [comment]
Thanks!

>BTW: According to the UDv2 guidelines, if you decide to use enhanced dependencies, you should copy all the basic dependencies to the DEPS columns except for the edges which you want to delete in the enhanced deps graph. It seems that you don't follow this rule and just add the edges which are enhanced-only.

This will be fixed in release v2.4. :)
